website/bot.py

```
@@ -1,202 +1,182 @@
+    db_folder_path = Path(db_folder_path)
+    temp_db_path = Path(temp_dir.name) / db_folder_path.name
+    temp_db_path.mkdir(parents=True, exist_ok=True)
+    return temp_dir, temp_db_path
+
+
+def embed_documents_and_save(embed_docs):
+    """Embed and save documents to FAISS index using OpenAI embeddings."""
+    db_folder_path = BASE_DIR / "faiss_index"
+    embeddings = OpenAIEmbeddings(model="text-embedding-3-small")
+
+    temp_dir, temp_db_path = get_temp_db_path(db_folder_path)
```

Ruff (0.14.7)

Unpacked variable `temp_db_path` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)